hamocc/mo_control_bgc.F90

```diff
@@ -48,34 +48,34 @@ module mo_control_bgc
   integer :: ldtbgc                   !  time step number from bgc restart file
   integer :: ldtrunbgc                !  actual time steps of run.
 
-  integer :: sedspin_yr_s = -1
-  integer :: sedspin_yr_e = -1
-  integer :: sedspin_ncyc = -1
-
   real    :: rmasks = 0.0             !  value at wet cells in sediment.
   real    :: rmasko = 99999.00        !  value at wet cells in ocean.
 
-  ! Logical switches set via namelist
-  logical :: l_3Dvarsedpor = .false.  ! apply lon-lat-depth variable sediment porosity via input file
-  logical :: do_ndep     =.true.      ! apply n-deposition
-  logical :: do_rivinpt  =.true.      ! apply riverine input
-  logical :: do_sedspinup=.false.     ! apply sediment spin-up
-  logical :: do_oalk     =.false.     ! apply ocean alkalinization
-  logical :: with_dmsph  =.false.     ! apply DMS with pH dependence
-
-  logical :: use_BROMO              = .false.
-  logical :: use_AGG                = .false.
-  logical :: use_WLIN               = .false.
-  logical :: use_natDIC             = .false.
-  logical :: use_CFC                = .false.
-  logical :: use_cisonew            = .false.
-  logical :: use_PBGC_OCNP_TIMESTEP = .false.
-  logical :: use_PBGC_CK_TIMESTEP   = .false.
-  logical :: use_FB_BGC_OCE         = .false.
-  logical :: use_BOXATM             = .false.
-  logical :: use_sedbypass          = .false.
+  ! Variables set via namelist bgcnml
+  logical           :: l_3Dvarsedpor          = .false. ! apply spatially variable sediment porosity
+  logical           :: do_ndep                = .true.  ! apply n-deposition
+  logical           :: do_rivinpt             = .true.  ! apply riverine input
+  logical           :: do_sedspinup           = .false. ! apply sediment spin-up
+  logical           :: do_oalk                = .false. ! apply ocean alkalinization
+  logical           :: with_dmsph             = .false. ! apply DMS with pH dependence
+  integer           :: sedspin_yr_s           = -1      ! start year for sediment spin-up
+  integer           :: sedspin_yr_e           = -1      ! end   year for sediment spin-up
+  integer           :: sedspin_ncyc           = -1      ! sediment spin-up sub-cycles
   character(len=64) :: ocn_co2_type
```

Comment:
One could set a comment, which values `ocn_co2_type` can hold (and what it is good for).